frontend/src/pages/rateLater/RateLater.tsx

```diff
@@ -45,6 +51,10 @@ const RateLaterPage = () => {
   const videoListTopRef = React.useRef<HTMLDivElement>(null);
   const [offset, setOffset] = React.useState(0);
   const limit = 20;
+  const settings = useSelector(selectSettings).settings?.[YOUTUBE_POLL_NAME];
```

Comment:
Make this generic.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1582/commits/202a03f534107e2904967568ca20a15788453819